### PR TITLE
Sys 831/pdf file processing script

### PR DESCRIPTION
**Pull request by kjallen**
This PR has a few related items wrapped in:

SYS-831 - Add text file processing 
SYS-832 - Add pdf file processing
SYS-871 - Add master and submaster copy operations (for audio, pdf and text. Image moved to separate ticket

To test, open bash shell on container:
`docker-compose exec django bash`
Run:
`python manage.py process_file -a 21198/zz002dx6v0 -f /home/django/dlcs-staff-ui/samples/example_project/sample.wav -g 315`
Expected result:
A file added to both:
`media_dev/oh_lz/oralhistory/audio/masters` and
`media_dev/oh_wowza/oralhistory/audio/submasters`
That follow the naming convention:
21198-zz002dx6v0-1-master.wav
21198-zz002dx6v0-1-submaster.mp3

The file sequence number in the file above may be different depending on your database, but the important item is that it is the same between files

The following can be repeated with similar results with both text and pdf samples:
`python manage.py process_file -a 21198/zz002dx6v0 -f /home/django/dlcs-staff-ui/samples/example_project/sample.pdf -g 315`
`python manage.py process_file -a 21198/zz002dx6v0 -f /home/django/dlcs-staff-ui/samples/example_project/sample.text -g 315`

Files should be added to:
`media_dev/oh_lz/oralhistory/pdf/masters` and
`media_dev/oh_lz/oralhistory/text/masters`

### CODE REVIEW

File: None/oralhistory/text/submasters/21198-zz002dx6v0-3-submaster..txt [hunk 1]
```diff
@@ -0,0 +1 @@
+sample
```
Comment (akohler) [inline None/oralhistory/text/submasters/21198-zz002dx6v0-3-submaster..txt:1]: @kjallen Please remove the `None` directory and contents from the repo.  I know it's created due to a bug, and I'll add a ticket for that.

File: oral_history/management/commands/process_file.py [hunk 2]
```diff
@@ -1,11 +1,13 @@
 import logging
 import mimetypes
 import os
+import pathlib
```
Comment (akohler) [inline oral_history/management/commands/process_file.py:4]: Happened to notice: in `process_file` this import is all of `pathlib` but only `pathlib.Path` is used.  In the various processor scripts like `audio_processor`, only `pathlib.Path` is imported (and used).  Please make these consistent.

File: oral_history/scripts/file_processor.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+import datetime
+import logging
+import mimetypes
+import os
+import shutil
+from django.core.management.base import CommandError
+from oral_history.models import ContentFiles
+from pathlib import Path
```
Comment (akohler) [inline oral_history/scripts/file_processor.py:10]: See previous comment on `pathlib.Path` in `process_file`.  Same uses, so please make these consistent - the specific import here is better than the broader one in `process_file`.

**Review by akohler — CHANGES_REQUESTED** [review]
@kjallen , just a few changes, all minor, then I'll approve & merge.
* Remove the `None` directory, we'll address the bug via separate ticket.
* Make imports consistent by changing the `Path`  instances in `process_file`.

Thanks for the test cases.  Here's what I got with the 3 runs (wav, pdf, txt):
```
akohler@DESKTOP-M03716M:~/projects/dlcs-staff-ui$ tree media_dev/
media_dev/
├── oh_lz
│   └── oralhistory
│       ├── audio
│       │   └── masters
│       │       └── 21198-zz002dx6v0-1-master.wav
│       ├── pdf
│       │   └── masters
│       │       └── 21198-zz002dx6v0-2-master.pdf
│       └── text
│           └── masters
│               └── 21198-zz002dx6v0-3-master.txt
├── oh_source
└── oh_wowza
    └── oralhistory
        └── audio
            └── submasters
                └── 21198-zz002dx6v0-1-submaster.mp3

13 directories, 4 files
```
The audios have matching sequences, then pdf and txt got new sequences, which I believe is expected.

File: oral_history/scripts/file_processor.py (hunk 3 above)
Comment (kjallen) [reply oral_history/scripts/file_processor.py:10]: Updated to use `from pathlib import Path` in all instances

File: oral_history/management/commands/process_file.py (hunk 2 above)
Comment (kjallen) [reply oral_history/management/commands/process_file.py:4]: Updated to use from pathlib import Path in all instances

**Pull request merged**